### PR TITLE
Feature/upgrade migration files

### PR DESCRIPTION
**Pull request by GustavoRSSilva**
This is ready to be merged, simplified and smother experience deploying the contracts. Trying to avoid to comment and change the contracts in the deploying phase.

screenshot from 2018-08-20 13-11-25

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -23,9 +23,18 @@ APPCOINSIAB_CONTRACT_NAME=appcoinsiab
 IAB_DEVELOPMENT_ADDRESS=address_appcoinsiab_development
```
Comment (DioPires) [inline .env.example]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

**Review by tcrosado — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (DioPires) [inline .env.example]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

**Pull request merged**